token/js/src/extensions/transferHook/instructions.ts

```diff
@@ -190,14 +192,120 @@ export async function addExtraAccountsToInstruction(
     return new TransactionInstruction({ keys: accountMetas, programId, data: instruction.data });
 }
 
+/**
+ * Construct an `ExecuteInstruction` for a transfer hook program
+ *
+ * @param programId             The program ID of the transfer hook program
+ * @param source                The source account
+ * @param mint                  The mint account
+ * @param destination           The destination account
+ * @param owner                 Owner of the source account
+ * @param validateStatePubkey   The validate state pubkey
+ * @param amount                The amount of tokens to transfer
+ * @returns Instruction to add to a transaction
+ */
+export function createExecuteInstruction(
+    programId: PublicKey,
+    source: PublicKey,
+    mint: PublicKey,
+    destination: PublicKey,
+    authority: PublicKey,
```

Comment:
micro-nit: we use the term `owner` everywhere else in the library, and even the comment for this function uses `owner`